dco-sample.ttl

```diff
@@ -0,0 +1,184 @@
+@prefix terms:   <http://purl.org/dc/terms/> .
+@prefix samfl:     <http://def.seegrid.csiro.au/ontology/om/sam-lite#> .
+@prefix dcosample:  <http://info.deepcarbon.net/sample/schema#> .
+@prefix vitro:   <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#> .
+@prefix rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dc:      <http://purl.org/dc/elements/1.1/> .
+@prefix owl:     <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms:  <http://purl.org/dc/terms/> .
+@prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vivo:    <http://vivoweb.org/ontology/core#> .
+@prefix obo:     <http://purl.obolibrary.org/obo/> .
+@prefix dco:     <http://info.deepcarbon.net/schema#> .
+@prefix xsd:     <http://www.w3.org/2001/XMLSchema#> .
+@prefix core:    <http://vivoweb.org/ontology/core#> .
+@prefix skos:    <http://www.w3.org/2004/02/skos/core#> .
+@prefix ero:     <http://purl.obolibrary.org/obo/> .
+@prefix bibo:    <http://purl.org/ontology/bibo/> .
+@prefix foaf:    <http://xmlns.com/foaf/0.1/> .
+
+<http://info.deepcarbon.net/sample/schema>
+      a       owl:Ontology ;
+      rdfs:label "DCO Sample"@en-US .
+
+dcosample:inSampleRepository
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:Sample ;
+      rdfs:label "in sample repository"@en-US ;
+      rdfs:range dcosample:SampleRepository ;
+      rdfs:comment  """
+      """^^rdf:XMLLiteral .
+
+dcosample:IGSNRegistration
+      a       owl:Class ;
+      rdfs:label "IGSN Registration"@en-US ;
+      rdfs:comment  """
+          information about the registration of an IGSN
+      """^^rdf:XMLLiteral .
+
+dcosample:igsnRegistrant
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:IGSNRegistration ;
+      rdfs:label "registered by"@en-US ;
+      rdfs:range foaf:Person ;
+      rdfs:comment  """
+          registration of the sample to get igsn was done by a person
+      """^^rdf:XMLLiteral .
+
+dcosample:includesSampleType
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:SampleGroup ;
+      rdfs:label "includes sample type"@en-US ;
+      rdfs:range dcosample:SampleType ;
+      rdfs:comment  """
+          A collection of samples includes samples of various types
+      """^^rdf:XMLLiteral .
+
+dcosample:hasSampleType
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:Sample ;
+      rdfs:label "has sample type"@en-US ;
+      rdfs:range dcosample:SampleType ;
+      rdfs:subPropertyOf core:hasSubjectArea ;
+      rdfs:comment  """
+          A sample classification or annotation
+      """^^rdf:XMLLiteral .
+
+dcosample:hasIGSNRegistration
```

Comment:
Should there be an inverse property for this property?

The IGSN Registration page has no link to the registered sample/feature https://udco.tw.rpi.edu/vivo/display/n16187